test/message/test_runner_output.js

```diff
@@ -383,3 +383,17 @@ test('unfinished test with unhandledRejection', async () => {
     setTimeout(() => Promise.reject(new Error('bar')));
   });
 });
+
+test('should print error cause', () => {
+  throw new Error('foo', {cause: new Error('bar')});
```

Comment:
Please add `Error.stackTraceLimit = 0;` to eliminate the stack traces from the output (and therefore the streams of asterisks) unless you’re intending to evaluate something in the stack trace. It’s annoying for tests to start failing because a refactoring changes the number of lines of a stack trace.